tree/ntuple/v7/inc/ROOT/RNTupleDescriptor.hxx

```diff
@@ -121,6 +121,11 @@ public:
    DescriptorId_t GetParentId() const { return fParentId; }
    const std::vector<DescriptorId_t> &GetLinkIds() const { return fLinkIds; }
    const std::vector<DescriptorId_t> &GetLogicalColumnIds() const { return fLogicalColumnIds; }
+
+   /// Tells if the field describes a user-defined class with a dictionary.
+   /// The dictionary does not need to be available for this method.
+   /// For unsplit fields, the method returns false independent of their content.
```

Comment:
Hm, why? Can we maybe handle them in the same way?

Reply:
Ah, I forgot about runtime polymorphism for unsplit fields. That's why we need the dynamic adding of streamer infos anyway...